functions/src/default.ts

```diff
@@ -113,14 +113,22 @@ If you can't get the PR to a green state due to flakes or broken master, please
     l1TriageLabels: [["comp: *"]],
     // arrays of labels that determine if a PR has been fully triaged
     l2TriageLabels: [["type: *", "effort*", "risk*", "comp: *"]]
-  }
+  },
+
+  rerunCircleCI: {
+    // set to true to disable
+    disabled: true,
+    // the label which when added triggers a rerun of the default CircleCI workflow.
+    triggerRerunLabel: 'Trigger CircleCI Rerun';
```

Comment:
this default label is too long and misleading. We are not just rerunning the CircleCI workflow, we are rerunning it  **rebased against the latest HEAD of the target branch**.

Maybe "Rerun CI against HEAD"?

Reply:
My concern with just saying CI is that not all things that show up on CI will be rerun, g3 presubmits being an example.

Reply:
I'll merge this for now, since this label will be part of the config on each repo anyway you can debate the name of the label on the PR that @josephperrott will make to update the config file